resolwe_bio/processes/workflows/bbduk_star_featurecounts_qc.py

```diff
@@ -0,0 +1,708 @@
+            input_star["alignment"][
+                "align_gap_max"
+            ] = inputs.alignment.alignment_options.align_gap_max
+
+        if inputs.alignment.output_options.out_rg_line:
+            input_star["output_options"][
+                "out_rg_line"
+            ] = inputs.alignment.output_options.out_rg_line
+
+        alignment = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input=input_star,
+        )
+
+        input_featurecounts = {
+            "aligned_reads": alignment,
+            "n_reads": inputs.quantification.n_reads,
+            "assay_type": inputs.assay_type,
+            "annotation": inputs.annotation,
+            "feature_class": inputs.quantification.feature_class,
+            "feature_type": inputs.quantification.feature_type,
+            "id_attribute": inputs.quantification.id_attribute,
+        }
+
+        if inputs.cdna_index:
+            input_featurecounts["cdna_index"] = inputs.cdna_index
+
+        quantification = Data.create(
+            process=BioProcess.get_latest(slug="feature_counts"),
+            input=input_featurecounts,
+        )
+
+        input_seqtk = {
+            "reads": preprocessing,
+            "n_reads": inputs.downsampling.n_reads,
+            "advanced": {
+                "seed": inputs.downsampling.advanced.seed,
+                "fraction": inputs.downsampling.advanced.fraction,
+                "two_pass": inputs.downsampling.advanced.two_pass,
+            },
+        }
+
+        if inputs.reads.type.startswith("data:reads:fastq:single:"):
+            slug_seqtk = "seqtk-sample-single"
+        elif inputs.reads.type.startswith("data:reads:fastq:paired:"):
+            slug_seqtk = "seqtk-sample-paired"
+        else:
+            self.error("Wrong input type.")
+
+        downsampling = Data.create(
+            process=BioProcess.get_latest(slug=slug_seqtk),
+            input=input_seqtk,
+        )
+
+        alignment_qc_rrna = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.rrna_reference,
+            },
+        )
+        alignment_qc_globin = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.globin_reference,
+            },
+        )
+
+        idxstats = Data.create(
+            process=BioProcess.get_latest(slug="samtools-idxstats"),
+            input={
+                "alignment": alignment,
+            },
+        )
+
+        multiqc_inputs = [
+            inputs.reads,
+            preprocessing,
+            alignment,
+            downsampling,
+            quantification,
+            alignment_qc_rrna,
+            alignment_qc_globin,
+            idxstats,
+        ]
+
+        inputs_multiqc = {
```

Comment:
Above chunk `multiqc_inputs` could be incorporated into this dict without an intermediate step as `multiqc_inputs` is not used anywhere else.

Renaming `inputs_multiqc` to `input_multiqc` would make it more in line with the rest of the input variable names used in this process.